core/fetcher.go

```diff
@@ -78,6 +79,11 @@ func (f *BlockFetcher) GetBlockByHash(ctx context.Context, hash libhead.Hash) (*
 	return res.Block, nil
 }
 
+// GetSignedBlock queries Core for a `Block` at the given height.
+func (f *BlockFetcher) GetSignedBlock(ctx context.Context, height *int64) (*coretypes.ResultSignedBlock, error) {
```

Comment:
why make this change in this PR? don't you only need it for core exchange PR?

Reply:
TestBlockFetcher_GetBlock_and_SubscribeNewBlockEvent is designed around checking results from sub against rpc call. I would either comment it out in this PR or combine both in one

Reply:
ack